europeana-entity-harvester/index.html

```diff
@@ -77,6 +78,17 @@
             return entityUrl.match(/\/(agent|concept|place|organization)\//)[1];
         }
 
+        function getEntityTypeHumanReadable(type) {
+          const names = {
+            agent: 'person',
+            concept: 'topic',
+            place: 'place',
+            organization: 'organization'
+          };
+          if (!type) return;
+          return names[type.toLowerCase()];
```

Comment:
What anticipated scenario is `.toLowerCase()` here to handle?

I think it's quite reasonable to expect `type` to always be in the correct case.

Reply:
Indeed, you are right, that doesn't make sense.